src/hivpy/hiv_testing.py

```diff
@@ -202,6 +202,8 @@ def test_mark_anc(self, pop, time_step):
                                                    (col.LAST_PREGNANCY_DATE, op.le, pop.date
                                                     - timedelta(days=270))])
             self.update_sub_pop_test_date(pop, third_trimester_pop, self.prob_anc_test_trim3)
+            # remove from antenatal care
+            pop.set_present_variable(col.ANC, False, third_trimester_pop)
```

Comment:
Is there somewhere we can put this that is more intuitive? It's not really a part of the testing logic per se

Reply:
I would prefer a cleanup function to be called from evolve!